docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -282,5 +282,5 @@ The same holds true for use-case-defined APIs that trigger automatic updates/cre
 This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
 
 - SPDX-License-Identifier: CC-BY-4.0
-- SPDX-FileCopyrightText: 2023 Contributors of the Eclipse Foundation
-- Source URL: [https://github.com/eclipse-tractusx/tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc)
+- SPDX-FileCopyrightText: 2023, 2024 Contributors of the Eclipse Foundation
```

Comment:
Hi @arnoweiss, you could also opt for only leaving the creation date. This way, you do not need to maintain the range on every change

Reply:
I actually like it this way as it also shows activity.